units/Legion/Legion EvoCom/legcomlvl4.lua

```diff
@@ -363,6 +363,9 @@ return {
 				weapontimer = 5,
 				weapontype = "MissileLauncher",
 				weaponvelocity = 650,
+				customparams = {
+					stockpilelimit = 3,
+				},
```

Comment:
Unused weapon but has stockpile configuration so should have limit, or could be deleted entirely

Reply:
It should be safe to remove the napalmmissile from the legion comms, it probably won't be needed in the future and wouldn't be difficult to remake.

Reply:
Removed unused weapondef